cloudinfo/resourcecontroller.go

```diff
@@ -66,6 +67,74 @@ func (infoSvc *CloudInfoService) ListResourcesByGroupID(resourceGroupId string)
 	return allResources, nil
 }
 
+func (infoSvc *CloudInfoService) GetReclamationIdFromCRN(CRN string) (string, error) {
```

Comment:
CloudInfoService does have an interface, in service.go line 56, so your three new public functions should be added to that interface (otherwise the PR tests will not be able to call them since they use the interface).